### ISSUE TITLE
Rounded caps for `Lines, LineScatters` make lines too long

### ISSUE THREAD

**Issue by fatteneder** [fatteneder/SVGMakie.jl#2]
`stroke-linecap = "round"` adds a half circle centered at the endpoint of the line.
Cf. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap

How do Cairo and GLMakie handle those? To get it right, we would have to shorten the line by half a stroke width on each side.

**Comment by asinghvi17** [comment]
I don't think we do any shortening in CairoMakie.  Found out in a Slack thread...
